pyproject.toml

```diff
@@ -1,3 +1,99 @@
+[build-system]
+requires = ["setuptools", "setuptools-scm"]
+build-backend = "setuptools.build_meta"
+
+[project]
+dynamic = ["version"]
+name = "arches"
+readme = "README.md"
+authors = [
+    {name = "Farallon Geographics, Inc", email = "dev@fargeo.com"}
+]
+license = {text = "GNU AGPL3"}
+requires-python = ">=3.10"
+description = "Arches is an open-source, web-based, geospatial information system for cultural heritage inventory and management."
+keywords = ["django", "arches", "cultural heritage"]
+classifiers = [
+    "Development Status :: 5 - Production/Stable",
+    "Intended Audience :: Developers",
+    "Intended Audience :: Science/Research",
+    "Intended Audience :: Information Technology",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3 :: Only",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+    "Programming Language :: Python :: 3.12",
+    "Framework :: Django",
+    "Framework :: Django :: 4.2",
+    "License :: OSI Approved :: GNU Affero General Public License v3 or later (AGPLv3+)",
```

Comment:
There was a slight discrepancy with this Trove classifier before -- used to say GPL3